From 421abc252b10c32ee087771a7bb622e645a60f2c Mon Sep 17 00:00:00 2001
From: Matthew <matthew@urgero.net>
Date: Thu, 6 Jun 2024 05:20:57 -0500
Subject: [PATCH 01/46] Update gitian descriptors (#2)

Builds failing due to `apt-get update` failure. Also updated the
`remotes` property in the YAML files to point to Lotusia Stewardship
repo.
---
 contrib/gitian-descriptors/gitian-linux.yml | 4 ++--
 contrib/gitian-descriptors/gitian-osx.yml   | 4 ++--
 contrib/gitian-descriptors/gitian-win.yml   | 4 ++--
 3 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/contrib/gitian-descriptors/gitian-linux.yml b/contrib/gitian-descriptors/gitian-linux.yml
index e048d9c5d..36e79ae9d 100644
--- a/contrib/gitian-descriptors/gitian-linux.yml
+++ b/contrib/gitian-descriptors/gitian-linux.yml
@@ -29,11 +29,11 @@ packages:
 - "python3"
 repositories:
 - "distribution": "buster-backports"
-  "source": "deb http://deb.debian.org/debian/ buster-backports main"
+  "source": "deb http://archive.debian.org/debian/ buster-backports main"
   packages:
   - "cmake"
 remotes:
-- "url": "https://github.com/LogosFoundation/lotusd.git"
+- "url": "https://github.com/LotusiaStewardship/lotusd.git"
   "dir": "bitcoin"
 files: []
 script: |
diff --git a/contrib/gitian-descriptors/gitian-osx.yml b/contrib/gitian-descriptors/gitian-osx.yml
index ae7b7517d..4ab8b35c1 100644
--- a/contrib/gitian-descriptors/gitian-osx.yml
+++ b/contrib/gitian-descriptors/gitian-osx.yml
@@ -31,11 +31,11 @@ packages:
 - "python3-setuptools"
 repositories:
 - "distribution": "buster-backports"
-  "source": "deb http://deb.debian.org/debian/ buster-backports main"
+  "source": "deb http://archive.debian.org/debian/ buster-backports main"
   packages:
   - "cmake"
 remotes:
-- "url": "https://github.com/LogosFoundation/lotusd.git"
+- "url": "https://github.com/LotusiaStewardship/lotusd.git"
   "dir": "bitcoin"
 files:
 - "Xcode-11.3.1-11C505-extracted-SDK-with-libcxx-headers.tar.gz"
diff --git a/contrib/gitian-descriptors/gitian-win.yml b/contrib/gitian-descriptors/gitian-win.yml
index eff8c71a1..ba2f73cad 100644
--- a/contrib/gitian-descriptors/gitian-win.yml
+++ b/contrib/gitian-descriptors/gitian-win.yml
@@ -25,11 +25,11 @@ packages:
   - "zip"
 repositories:
   - "distribution": "buster-backports"
-    "source": "deb http://deb.debian.org/debian/ buster-backports main"
+    "source": "deb http://archive.debian.org/debian/ buster-backports main"
     packages:
       - "cmake"
 remotes:
-  - "url": "https://github.com/LogosFoundation/lotusd.git"
+  - "url": "https://github.com/LotusiaStewardship/lotusd.git"
     "dir": "bitcoin"
 files: []
 script: |
-- 
2.39.5 (Apple Git-154)

